src/main/java/parking/services/PricingService.java

```diff
@@ -0,0 +1,42 @@
+package parking.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import parking.entities.DateContainer;
+import parking.entities.Plate;
+
+import java.rmi.activation.UnknownObjectException;
```

Comment:
Czemu akurat taki wyjątek? Co to jest RMI?

Reply:
j/w. Problem leżał w tym, że chciałem pokryć sytuację, w której użyto wartości nie znajdującej się enumie Region. W sumie IllegalArgumentException by się lepiej nadawał.
RMI - Remote Method Invocation.

Reply:
Chodziło mi o to, że nie powinieneś używać tej klasy wyjątku, ponieważ jest z pakietu java.rmi.activation